cou/utils/juju_utils.py

```diff
@@ -136,7 +142,7 @@ async def _async_get_current_model_name_from_juju():
     return model_name
 
 
-async def async_get_status(model_name=None) -> FullStatus:
+async def async_get_status(model_name: Optional[str] = None) -> FullStatus:
     """Return the full juju status output.
 
     :param model_name: Name of model to query.
```

Comment:
change `:type model_name: str` to `:type model_name: Optional[str]`

Reply:
ok